e2e/tests/notebook/notebook-reader-actions.spec.ts

```diff
@@ -55,7 +55,7 @@ describe('Workspace READER Jupyter notebook action tests', () => {
   });
 
   // TODO(RW-7312): update and re-enable
-  test.skip('Workspace READER copy notebook to another workspace', async () => {
+  test('Workspace READER copy notebook to another workspace', async () => {
```

Comment:
The  notebook-reader test is failing at this line [110](https://github.com/all-of-us/workbench/pull/6413/files#diff-7938c6cd0f112f053d880ae1629e479a3ec3ec41958eb93591f2451bc8381e43L106-L110) & [209](https://github.com/all-of-us/workbench/pull/6413/files#diff-7938c6cd0f112f053d880ae1629e479a3ec3ec41958eb93591f2451bc8381e43L205-L209). 
On commenting out these lines the test is passing.  
@aweng98 you might have to check `notebook-preview-utils.ts` file, I could not figure out.